### ISSUE TITLE
sk-resnet50

### ISSUE THREAD

**Issue by Yao-DD** [implus/PytorchInsight#24]
As paper(Selective Kernel Networks) said，the conventional convolution with a 5×5 kernel is replaced with the dilated convolution with a 3×3 kernel and dilation size 2. But the code is implemented as:
"
 self.conv2g = conv3x3(planes, planes, stride, groups = 32)
 self.bn2g   = nn.BatchNorm2d(planes)
"
I can't understand why. Could you tell me? thank you.

**Comment by jianghaojun** [comment]
I found another mistake: self.conv2 is not implemented as groupconv. 

`self.conv2 = conv3x3(planes, planes, stride)`
`self.bn2 = nn.BatchNorm2d(planes)`
`self.conv2g = conv3x3(planes, planes, stride, dilation=2, groups = 32)`
`self.bn2g   = nn.BatchNorm2d(planes)`

And I checked the [sknet50.prototxt ](https://github.com/implus/SKNet/blob/master/models/sknet50.prototxt) in original caffe repo, both of the conv3x3s' groups are 32:
[conv2_1/3x3g32](https://github.com/implus/SKNet/blob/17dd7086d4959be0c8c6ccce52e4fd8187836770/models/sknet50.prototxt#L150)
[conv2_1/3x3g32d2](https://github.com/implus/SKNet/blob/17dd7086d4959be0c8c6ccce52e4fd8187836770/models/sknet50.prototxt#L211)
